lib/perl/Genome/Model/ReferenceAlignment/Command/GenotypeMicroarrayConcordance.pm

```diff
@@ -19,10 +19,16 @@ class Genome::Model::ReferenceAlignment::Command::GenotypeMicroarrayConcordance
             doc => 'The minimum depth required to consider concordant.',
             example_values => ['4'],
         },
+        bedtools_version => {
+            is => 'Text',
+            doc => 'Version of bed tools intersect to use.',
+            default_value => Genome::Model::Tools::BedTools->latest_bedtools_version,
```

Comment:
If its run by hand, I think a default value is fine for now with the caveat that we should probably remove it should it become part of an automated process!